docs/usage/custom-device-files.md

```diff
@@ -1,12 +1,23 @@
-# Testing custom devices config
+# Using custom devices configuration files
```

Comment:
I think we should have one section for device files, this together with adding/requesting device support.

Also we should make it very clear that an "unknown" device doesn't automatically mean it is not supported - due to how Z-Wave works.